blueprints/mixin-test/index.js

```diff
@@ -5,7 +5,8 @@ var testInfo = require('ember-cli-test-info');
 module.exports = {
   description: 'Generates a mixin unit test.',
   locals: function(options) {
-    return {
+  	return {
```

Comment:
good catch. argh tabs vs spaces... I had the wrong setting on this new computer.